slices/gsettings-desktop-schemas.yaml

```diff
@@ -0,0 +1,7 @@
+package: gsettings-desktop-schemas
+
+slices:
+  data:
+    contents:
+      /usr/share/GConf/gsettings/*:
+      /usr/share/glib-2.0/schemas/*:
```

Comment:
same here. I believe this whole slice should be called `config`. `data` is more appropriate for something like `ca-certificates` which delivers data files like `/etc/ssl/certs/ca-certificates.crt`.

Reply:
these are all ascii files, so let's keep data as a catch all